app/styles/all.scss

```diff
@@ -9,6 +9,10 @@ $color_nhsuk-white--alpha20: rgba($color_nhsuk-white, .2);
 $color_nhsuk-white--alpha30: rgba($color_nhsuk-white, .3);
 $color_nhsuk-white--alpha40: rgba($color_nhsuk-white, .4);
 
+.bcc-t-font-color-grey {
```

Comment:
please can you rename this `bcc-t-font-grey1` as the nhs have about 6 different greys